data/resources/ui/content-message-media-content.ui

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<interface>
+  <template class="ContentMessageMediaContent" parent="GtkWidget">
```

Comment:
I like the "Content" at the end because the widget is literally the MessageMedia's content property. But I agree that the prefix makes it weird, but honestly I think the problem is the naming scheme we're using that is a bit weird. I was thinking of dropping all the directory-based prefixes and just using "Telegrand", like any other gtk app. E.g. this would be "TelegrandMessageMediaContent" with that change. But that's something to be done separately.